plugins/mcreator-core/datalists/itemproperties.yaml

```diff
@@ -0,0 +1,26 @@
+- damaged:
+  type: Logic
+  other:
+    min: 0
+    max: 1
+- damage:
+  type: Number
+  other:
+    min: 0
+    max: 1
+- lefthanded:
+  type: Logic
+  other:
+    min: 0
+    max: 1
+- cooldown:
+  type: Number
+  other:
+    min: 0
+    max: 1
+- custom_model_data

```

Comment:
This one does not seem to be used or make sense to be in a mapping so I would remove it

Reply:
It is here in this format to not let users define another property with this name and at the same time prevent them from using it in custom states (my thoughts are this property is reserved for custom data packs only).